mtools/mlaunch/mlaunch.py

```diff
@@ -1537,7 +1537,8 @@ def _filter_valid_arguments(self, arguments, binary="mongod",
             elif i > 0 and arguments[i - 1] in result:
                 # if it doesn't start with a '-', it could be the value of
                 # the last argument, e.g. `--slowms 1000`
-                result.append(arg)
+                # NB: arguments are always quoted
+                result.append('"%s"' % arg)
```

Comment:
LGTM, but since this release will be Python 3.6+, we might want to use f-strings going forward. They're fast, flexible, and readable:

`result.append(f'"{arg}"')`

Reply:
Great suggestion. Thanks @kallimachos!